includes/wc-formatting-functions.php

```diff
@@ -271,11 +271,11 @@ function wc_format_decimal( $number, $dp = false, $trim_zeros = false ) {
 
 	// DP is false - don't use number format, just return a string in our format
 	} elseif ( is_float( $number ) ) {
-		$number = wc_clean( str_replace( $decimals, '.', strval( $number ) ) );
+		$number = strval( $number );
```

Comment:
When you convert a float to string in PHP, PHP uses the decimal for the LOCALE, not '.'. This enforces '.' for DB storage.